CHANGELOG.md

```diff
@@ -6,11 +6,31 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic
 Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.3.1] - 2022-01-04
+
+We recommend Clang >= 10.0 as the host compiler to avoid false-positive
+deprecation warnings.
+
+### Fixed
+
+- Remove blanket-statement error message upon early buffer deallocation, which
+  in many cases is now legal. (6851145)
+- Properly apply horizons to collective host task order-dependencies. (4488724)
+- Avoid race between horizon task generation and horizon command execution.
+  (f670868)
+- Fix data race in `task_manager::notify_horizon_executed`. (f641bc)
```

Comment:
I don't know if it needs to be clarified that this was in debug builds.

Reply:
Done!